src/components/modeler/Modeler.vue

```diff
@@ -880,6 +880,15 @@ export default {
             }
           });
         }
+        let laneSets = node.definition.processRef.laneSets;
+        if (laneSets != null && laneSets.length > 0) {
```

Comment:
```suggestion
        if (laneSets !== null && laneSets.length > 0) {
```

Reply:
Hey @josechirivella, thanks for the review. I did not test the type to avoid getting back to it in case `lanseSets` is set to `undefined` instead of `null`. What do you think?

Reply:
If we are worried for the value to be of falsy type such as `null` `undefined` or `false` we can always do `!!`
https://betterprogramming.pub/javascript-bang-bang-i-shot-you-down-use-of-double-bangs-in-javascript-7c9d94446054
such as
```suggestion
        if (!!laneSets && laneSets.length > 0) {
```